x-pack/test_serverless/functional/services/svl_search_navigation.ts

```diff
@@ -47,5 +47,13 @@ export function SvlSearchNavigationServiceProvider({
       });
       await testSubjects.existOrFail('searchIndicesDetailsPage', { timeout: 2000 });
     },
+    async navigateToInferenceManagementPage(expectRedirect: boolean = false) {
+      await PageObjects.common.navigateToApp('searchInferenceEndpoints', {
+        shouldLoginIfPrompted: false,
+      });
+      if (!expectRedirect) {
```

Comment:
:x: you don't need this `expectRedirect` that was specifically added to `navigateToElasticsearchStartPage` since if you have indices it redirects to the index management page. but theres not reason to copy that here.